chihacknight/decarbonize-my-state
# responsive markup for power plant icons, added plant name

## Overview

Some mobile and design tweaks to the power plant image list completed in #67 

* reduces the count of plants to 10
* adds the name of each plant under the image

@vkoves I could use an assist from you on aligning the "... and XX more" text properly. And of course any other mobile / accessibility changes you think are necessary.

### Demo

Default view
![Screen Shot 2022-04-29 at 12 00 09 PM](https://user-images.githubusercontent.com/919583/165990431-d4d3b148-9c77-4cd6-a5c2-47d298d57dd3.png)

Mobile
![Screen Shot 2022-04-29 at 12 00 20 PM](https://user-images.githubusercontent.com/919583/165990453-9bee5676-12f5-45bc-a5fb-0df214562394.png)

## Testing Instructions

* view the power plants section on mobile and desktop and confirm layout looks good and scrolling isn't too arduous
